### ISSUE TITLE
TinyMCE 5: link_context_toolbar does not show external link icon

### ISSUE THREAD

**Issue by davideickhoff** [tinymce/tinymce#4895]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
setting `link_context_toolbar: true` for the link plugin does not show icon to open link.

**What is the expected behavior?**
The docs say:
```
Setting link_context_toolbar to true will enable context toolbar on every link, with shortcut functions to open the link, modify it or - remove. External links will be opened in a separate tab, while relative ones will cause scroll to a target within the editor (if the target is found).
```

Opening external links is currently not possible.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
did work with 4x, does not work with 5.0.2 on Chrome (didn't test other browser)

**Comment by lnewson** [comment]
Thanks for the report! I can confirm this is missing and I checked with the team and it wasn't intentionally removed, so we'll try and get this back as soon as we can.

**Comment by wonderbugtiny** [comment]
This should now be fixed in 5.0.3. Please let us know if you find any further problems!
